### PR TITLE
Update sites.html

### PR DESCRIPTION
**Pull request by Anthony-B-Russo10**
Update sites.html to include more help Celestia Links.

### CODE REVIEW

**Comment by 375gnu** [comment]
only the site and the forum are "official" resources. i don't think it's a good idea to list all 3rd party resources here.

**Comment by TheLostProbe** [comment]
im unsure on the celestia.mobi source code one. but most, if not all the others should be kept, especially the wikibooks since it is good for first-timers and people looking to troubleshoot. social platforms such as the subreddit and the discord should probably also be kept since some people may not know how to use the forums, and response times on discord are much faster because it is real-time. the archive repository is good for people looking for old versions of celestia, since the sourceforge is now practically obsolete

**Comment by levinli303** [comment]
the celestia.mobi repository does not contain critical source code (it only has css stylesheets and some redirection links) so it is unnecessary to put it up there.

**Comment by Anthony-B-Russo10** [comment]
I mainly put the celestia.mobi repository there due to that being where the addon submissions are sent for celestia.mobi's addons section though if need be I'll change the link to go to the repository's Issues tab.

**Comment by levinli303** [comment]
we can just put https://celestia.mobi since the front page has a link to submission.

**Comment by 375gnu** [comment]
I'm okay with https://celestia.mobi/ addition, but not other resources

**Comment by 375gnu** [comment]
agree with wikibooks.

**Comment by Anthony-B-Russo10** [comment]
Updated to change Mobi's link to the main page.

> i think reddit and discord links should be added to the footer alongside the twitter, github, and email, instead of in sites.html.

Those would also need to be updated to all the html files for the site, but yeah that would be a better place for those.

**Comment by TheLostProbe** [comment]
problem is that i dont know where the icon files for the footer links are stored. im guessing they are stored somewhere else on the server and not in the websites files itself, or perhaps they were not added to the github repository

**Comment by Anthony-B-Russo10** [comment]
I think they are all stored on the server side of the site, but what is odd is that I can't even find the icons when doing an inspect element on the site as well.

File: sites.html [hunk 1]
```diff
@@ -83,7 +83,24 @@ <h2>Related Sites</h2>
                                 	Each of the tours comes with documentation and a student worksheet for
                                 	classroom use.
                                 	<br><br></li>
-                                	<li><b><a href="/ru/forum" target="_blank">Celestia Russian Forum</a></b><br>
+					<br><br></li>
+					<li><b><a href="/ru/forum" target="_blank">Old Celestia Russian Forum</a></b><br>
+						Russian-language Celestia Forums. Now obsolete and read-only, only kept up because of resources avaliable there.
+					br><br></li>
+					<li><b><a href="https://discord.com/invite/WEWDcJh" target="_blank">Celestia Official Discord Server</a></b><br>
```
Comment (375gnu) [inline sites.html:90]: remove this

File: sites.html [hunk 2]
```diff
@@ -83,7 +83,24 @@ <h2>Related Sites</h2>
...
+						A Discord community for Celestia created by Lafuente_Astronomy.
+                                	<br><br></li>
+					<li><b><a href="https://www.reddit.com/r/Celestiasoftware/" target="_blank">Celestia Official Subreddit</a></b><br>
```
Comment (375gnu) [inline sites.html:93]: remove this

File: sites.html [hunk 3]
```diff
@@ -83,7 +83,24 @@ <h2>Related Sites</h2>
...
+                                	A subreddit for Celestia created by Anthony_B_Russo10.
+                                	<br><br></li>
+					<li><b><a href="https://en.wikibooks.org/wiki/Celestia" target="_blank">Celestia Wikibooks</a></b><br>
+                                	Celestia online Wikibooks for help documentation.
+                                	<br><br></li>
+					<li><b><a href="https://celestia.mobi/" target="_blank">Celestia.mobi</a></b><br>
+                                	Celestia.mobi main site with links to downloads of the mobile versions of Celestia along with hosted addons.
+                                	<br><br></li>
+					<li><b><a href="https://github.com/Anthony-B-Russo10/Celestia-Archive" target="_blank">Celestia Archive GitHub Repository</a></b><br>
```
Comment (375gnu) [inline sites.html:102]: remove this

File: sites.html [hunk 4]
```diff
@@ -83,7 +83,24 @@ <h2>Related Sites</h2>
...
+						A Discord community for Celestia created by Lafuente_Astronomy.
+                                	<br><br></li>
+					<li><b><a href="https://www.reddit.com/r/Celestiasoftware/" target="_blank">Celestia Official Subreddit</a></b><br>
+                                	A subreddit for Celestia created by Anthony_B_Russo10.
+                                	<br><br></li>
+					<li><b><a href="https://en.wikibooks.org/wiki/Celestia" target="_blank">Celestia Wikibooks</a></b><br>
+                                	Celestia online Wikibooks for help documentation.
+                                	<br><br></li>
+					<li><b><a href="https://celestia.mobi/" target="_blank">Celestia.mobi</a></b><br>
+                                	Celestia.mobi main site with links to downloads of the mobile versions of Celestia along with hosted addons.
```
Comment (375gnu) [inline sites.html:100]: plz change to "Celestia for mobile devices"

**Review by 375gnu — CHANGES_REQUESTED** [review]

**Comment by Anthony-B-Russo10** [comment]
Changes performed.

**Review by 375gnu — CHANGES_REQUESTED** [review]
please squash commits

**Comment by Anthony-B-Russo10** [comment]
One problem, Github won't let me squash the commits.
image

**Comment by Anthony-B-Russo10** [comment]
Commits squashed.

**Comment by 375gnu** [comment]
There are still 4 commits instead of 1.

**Comment by Anthony-B-Russo10** [comment]
Squashed

**Comment by 375gnu** [comment]
image

**Comment by Anthony-B-Russo10** [comment]
One problem.
image

**Comment by TheLostProbe** [comment]
you should be able to do this much easier through github desktop or using git

**Comment by Anthony-B-Russo10** [comment]
Only way I figured out how to do it was on GitHub desktop and even that doesn't want to do it now.

**Comment by 375gnu** [comment]
git rebase -i master

**Comment by Anthony-B-Russo10** [comment]
This leads to problem number two, I have never used GitHub via command line.

**Comment by 375gnu** [comment]
git push -f

**Comment by Anthony-B-Russo10** [comment]
That still doesn't work.
image

**Comment by 375gnu** [comment]
git remote set-url git@github.com:CelestiaProject/www.git

and upload your ssh key to gh account

**Comment by Anthony-B-Russo10** [comment]
The SSH key I uploaded last night, that didn't do much to help.
And:
image

**Comment by 375gnu** [comment]
git remote set-url origin git@github.com:CelestiaProject/www.git

**Comment by Anthony-B-Russo10** [comment]
I got it down to two commits, I can not get it any lower as not Git will not connect to the origin. The only way this is getting down to one commit is by creating a new pull request with the same changes.

**Comment by 375gnu** [comment]
please create a new PR then

**Pull request closed without merging**